brayns/Brayns.cpp

```diff
@@ -138,15 +138,14 @@ struct Brayns::Impl
         if(!geometryParameters.getXYZBFile().empty())
             _loadXYZBFile();
 
-        if(!volumeParameters.getFilename().empty())
+        if(!volumeParameters.getFilename().empty() || !volumeParameters.getFolder().empty())
         {
-            const Vector3ui& volumeDimensions = scene->getVolumeHandler()->getDimensions();
-            const Vector3f& volumeScale = volumeParameters.getScale();
+            const Vector3ui& volumeDimensions = scene->getVolumeHandler()->getDimensions( 0.f );
```

Comment:
Still there. It feels wrong. All the methods in VolumeHandler take the timestamp as an argument. Since it already is a member, couldn't you have a setTimeStamp method, and then call the getter (without argument) for the volume that is currently set?